### PR TITLE
[Add] support hermes in code push

### PR DESCRIPTION
**Pull request by phu-tang-tiki**
I see that the appcenter codepush cli currently support hermes on codepush. So I make this PR to add this parameter on fastlane function. 

Please review and let me know if need to improve

### CODE REVIEW

**Comment by DmitriyKirakosyan** [comment]
@phu-tang-tiki , thank you for the contribution!
The argument `use-hermes` is used to bypass the hermes check. In normal scenario you don't need to use this argument, and hermes will be automatically enabled if it is set in your project configuration. However in some cases you would need to bypass the check due to specific project configuration. Is it the case why you are adding this parameter?

**Comment by phu-tang-tiki** [comment]
yes, I need to use that parameter, specially for Android version. Let say we custom a lot in build flavor and variants and have to enable it manually.

File: lib/fastlane/plugin/appcenter/actions/appcenter_codepush_release_react.rb [hunk 1]
```diff
@@ -74,6 +75,9 @@ def self.run(params)
         if xcode_project_file
           command += "--xcode-project-file #{xcode_project_file} "
         end
+        if use_hermes
+          command += "--use-hermes"
```
Comment (DmitriyKirakosyan) [inline lib/fastlane/plugin/appcenter/actions/appcenter_codepush_release_react.rb:79]:
```suggestion
          command += "--use-hermes "
```

Comment (DmitriyKirakosyan) [reply lib/fastlane/plugin/appcenter/actions/appcenter_codepush_release_react.rb:79]: What if we wan't to bypass disabling hermes? I think it should be
```
command += "--use-hermes #{use_hermes} "
```

Comment (phu-tang-tiki) [reply lib/fastlane/plugin/appcenter/actions/appcenter_codepush_release_react.rb:79]: you're right, it should be `--use-hermes #{use_hermes}` let me update

Comment (phu-tang-tiki) [reply lib/fastlane/plugin/appcenter/actions/appcenter_codepush_release_react.rb]: added check and add some test for sure

**Comment by phu-tang-tiki** [comment]
@microsoft-github-policy-service agree

**Review by DergachevE — APPROVED** [review]

**Comment by dungpasoftware** [comment]
> yes, I need to use that parameter, specially for Android version. Let say we custom a lot in build flavor and variants so I have to enable it manually.

do we have parameters to custom flavor for android? I try to use appcenter_codepush_release_react, but don't find any parameter to custom flavor for code push

**Pull request merged**